### PR TITLE
More updates from the ATT review

### PR DESCRIPTION
**Pull request by dbelcham**
More tests moved to new locations and one file split for single-test-per-file

### CODE REVIEW

File: src/NServiceBus.AcceptanceTests/ScenarioDescriptors/TypeScanner.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@ namespace NServiceBus.AcceptanceTests.ScenarioDescriptors
     using System.Collections.Generic;
     using System.Linq;
     using System.Reflection;
-    using NServiceBus.Hosting.Helpers;
+    using Hosting.Helpers;
```
Comment (danielmarbach) [inline src/NServiceBus.AcceptanceTests/ScenarioDescriptors/TypeScanner.cs]: Is this change related?

Comment (timbussmann) [inline src/NServiceBus.AcceptanceTests/ScenarioDescriptors/TypeScanner.cs]: I think because we removed the hosting folder which removes the need to fully qualify the namespace in this case.
